pkg/privilege/privileges/cache.go

```diff
@@ -1331,6 +1331,14 @@ func (p *MySQLPrivilege) decodeColumnsPrivTableRow(userList map[string]struct{})
 			old.username = value.User
 		}
 		old.data = append(old.data, value)
+		logutil.BgLogger().Info("create column privilege record in cache",
```

Comment:
If you add log for create column privilege record in cache，should we also add revoke part log？

Reply:
Revoke is operated on system table. After modifying system table, a cache rebuild will be triggered.